github_nonpublic_api/api.py

```diff
@@ -104,15 +102,20 @@ def _create_org_callback(data):
                 data['organization[company_name]'] = business_name
 
         _get_and_submit_form(session=self._session,
-                                url=_CREATE_ORG_URL, data_callback=_create_org_callback,
-                                form_id='org-new-form')
+                             url=_CREATE_ORG_URL, data_callback=_create_org_callback,
+                             form_matcher=lambda form: form.attrib.get('id') == 'org-new-form')
+
+    def install_application_in_organization(self, app_name: str, org_id: int):
+        """Installs the specified app on the given organization."""
+        url = _INSTALL_APP_URL.format(app_name=app_name, org_id=org_id)
+
+        _get_and_submit_form(session=self._session,
+                             url=url,
+                             form_matcher=lambda form: app_name in form.attrib.get('action'))
```

Comment:
How sure are you that `app_name` is correct? I feel like I've seen `app_id` in URLs before, too.

Reply:
Yeah, they are lovingly inconsistent.  I suspect in the UI they end up using whatever is convenient, vs. any specific conventions.  I've seen examples where they use the app name in one URL and the app id in the next one....